tests/end_to_end/pytest.ini

```diff
@@ -0,0 +1,12 @@
+[pytest]
+addopts = -ra -q -s --junitxml=results/results.xml
+testpaths = test_suites
+junit_family = xunit2
+results_dir = results
+log_level = INFO
+markers =
+    torch_cnn_mnist: mark a test as a torch CNN MNIST test.
+    keras_cnn_mnist: mark a test as a Keras CNN MNIST test.
+    torch_cnn_histology: mark a test as a torch CNN histology test.
+    tf_2dunet: mark a test as a tf 2D U-Net test.
+    tf_cnn_histology: mark a test as a tf CNN histology test.
```

Comment:
Request removing TF tests. TF tutorials will also be dispositioned soon from OpenFL. Going forward, it will be Keras (with all backends), PyTorch and Jax.

Reply:
Removed tf based tests.